pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -34,6 +34,37 @@
   hoc_overview_oceans_tutorial_header: 'AI For Oceans'
   hoc_overview_oceans_tutorial_desc: 'Learn about AI, machine learning, training data, and bias, while exploring ethical issues and how AI can be used to address world problems.'
 
+  activity: 'Activity'
+  name: 'Name'
+  description: 'Description'
+  audience: 'Audience'
```

Comment:
These are pretty generic - is there any chance they'll conflict with existing keys?